docs/src/catalyst_applications/bifurcation_diagrams.md

```diff
@@ -1,115 +1,118 @@
-                        p_min = p_span[1],      # Min p-val (if hit, the method stops).
-                        p_max = p_span[2],      # Max p-val (if hit, the method stops).
-                        detect_bifurcation = 3) # Value in {0,1,2,3}
-nothing   # hide
+p_span = (2.0, 20.0)
+opt_newton = NewtonPar(tol = 1e-9, max_iterations = 1000)
+opts_br = ContinuationPar(p_min = p_span[1], p_max = p_span[2], 
+                          dsmin = 0.001, dsmax = 0.01, max_steps = 1000,
+                          newton_options = opt_newton)
+bif_dia = bifurcationdiagram(bprob, PALC(), 2, (args...) -> opts_br; bothside=true)
+nothing # hide
 ```
-Here `detectBifurcation` determines to what extent bifurcation points are
-detected and how accurately their values are determined. Three indicates to use the most
-accurate method for calculating their values.
+(however, in this case these additional settings have no significant effect on the result)
 
-We are now ready to compute the bifurcation diagram:
+## Bifurcation diagrams with disjoint branches
+Let's consider the previous case, but instead compute the bifurcation diagram over the interval *(2.0, 15.0)*:
 ```@example ex1
-bf = bifurcationdiagram(bprob, PALC(), 2, (args...) -> bopts)
-nothing   # hide
+p_span = (2.0, 15.0)
+opts_br = ContinuationPar(p_min = p_span[1], p_max = p_span[2], max_steps = 1000)
+bif_dia = bifurcationdiagram(bprob, PALC(), 2, (args...) -> opts_br; bothside=true)
+plot(bif_dia; xguide="k1", yguide="X")
 ```
-Finally, we can plot it:
-```@example ex1
-plot(bf; xlabel = string(bif_par), ylabel = string(plot_var))
+Here, in the bistable region, we only see a single branch. The reason is that the continuation algorithm start at our initial guess (here made at *k1 = 4.0* for *(X,Y) = (5.0,2.0)*) and tracks the diagram from there. However, with the upper bound set at *k1=15.0* the bifurcation diagram have a disjoint branch structure, preventing the full diagram from being computed by continuation alone. In this case it could be solved by increasing the bound of *k1=15.0*, however, this is not possible in all cases. In these cases, *deflation* can be used. This is described in the [BifurcationKit documentation](https://bifurcationkit.github.io/BifurcationKitDocs.jl/dev/tutorials/tutorials2/#Snaking-computed-with-deflation).
+
+
+## Systems with conservation laws
+Some systems are under-determined, and for a given parameter set they have an infinite number of possible steady states, preventing bifurcation diagrams from being computed. Similar to when we [compute single steady states](@ref homotopy_continuation_conservation_laws), we can utilise Catalyst's ability to detect and eliminate conservation laws to resolve this issue. This requires us to provide information of the species concentrations at which we wish to compute the bifurcation diagram. These are provided to the `BifurcationProblem` using the `u0` argument.
```

Comment:
```suggestion
Some systems are under-determined determined at steady-state, so that for a given parameter set they have an infinite number of possible steady state solutions, preventing bifurcation diagrams from being computed. Similar to when we [compute steady states for fixed parameter values](@ref homotopy_continuation_conservation_laws), we can utilise Catalyst's ability to detect and eliminate conservation laws to resolve this issue. This requires us to provide information of the species concentrations at which we wish to compute the bifurcation diagram (to determine the values of conserved quantities). These are provided to the `BifurcationProblem` using the `u0` argument.
```